server/graphql/v2/input/AccountInput.js

```diff
@@ -0,0 +1,35 @@
+import { GraphQLInt, GraphQLString, GraphQLInputObjectType } from 'graphql';
+
+import models from '../../../models';
+import { idDecode } from '../identifiers';
+import { NotFound } from '../../errors';
+
+const AccountInput = new GraphQLInputObjectType({
+  name: 'AccountInput',
+  fields: () => ({
+    id: { type: GraphQLString },
+    slug: { type: GraphQLString },
+    legacyId: { type: GraphQLInt },
+  }),
+});
+
+const fetchAccountWithInput = async input => {
```

Comment:
Ideally we should accept the `throwIfMissing` here. Sending this arg from frontend is not very nice, it was introduced because fetching a collective that slug doesn't exist in a `collective` query should return `null`, not an error but we didn't want to break the existing. 

In mutations however it's ok to throw if the provided collective doesn't exist.

So for new endpoints we'll normally won't have to expose the `throwIfMissing` param publicly.